apps/penxle.com/src/lib/components/forms/SelectOption.svelte

```diff
@@ -0,0 +1,29 @@
+<script lang="ts">
+  import { getContext } from 'svelte';
+  import type { Writable } from '@svelte-kits/store';
+
+  export let value: string;
+
+  let currentValue = getContext<Writable<string>>('currentValue');
+  let currentText = getContext<Writable<string>>('currentText');
+  let close = getContext<() => void>('close');
+</script>
+
+<li>
+  <button
+    class="body-14-sb text-secondary px-4 py-3 w-full rounded-lg hover:(bg-primary text-primary)"
```

Comment:
P3: 선택된 옵션에 대해 배경 스타일을 추가해주는게 어떨까요? 스타일링시 [aria-selected](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-selected) 어트리뷰트를 셀렉터로 삼아 접근성도 같이 챙기면 좋을 것 같습니다.

```suggestion
    class="body-14-sb text-secondary px-4 py-3 w-full rounded-lg hover:(bg-primary text-primary) aria-selected:(bg-primary text-primary)"
    aria-selected={currentValue === value}
```